core/src/main/java/org/sonarsource/sonarlint/core/container/storage/StoragePaths.java

```diff
@@ -121,6 +115,16 @@ public Path getActiveRulesPath(String qProfileKey) {
     return getGlobalStorageRoot().resolve(ACTIVE_RULES_FOLDER).resolve(encodeForFs(qProfileKey) + ".pb");
   }
 
+  private static String bytesToHex(byte[] bytes) {
```

Comment:
This looks very close to `org.sonarsource.sonarlint.core.tracking.DigestUtils#encodeHexString`, and a complete duplicate of `org.sonarsource.sonarlint.core.container.UidFactoryFast#byteArrayToHex`. Is there a particular reason for this?

Reply:
I was not remembering those methods existed :) Good catch!